### PR TITLE
Add support for Arch Linux

### PR DESCRIPTION
**Pull request by Barthalion**

### CODE REVIEW

**Comment by Barthalion** [comment]
I forgot about updating test cases, will do that shortly.

**Comment by Barthalion** [comment]
Force pushed updated patch.

File: spec/classes/openvpn_install_spec.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@
       :operatingsystemmajrelease => operatingsystemmajrelease,
       :operatingsystemrelease => operatingsystemrelease,
       :operatingsystem => operatingsystem,
-      :operatingsystemrelease => operatingsystemrelease,
```
Comment (luxflux) [inline spec/classes/openvpn_install_spec.rb]: :+1:

**Comment by luxflux** [comment]
Thank you for your work! :beers:

**Pull request merged**